internal/flink/command_artifact.go

```diff
@@ -35,8 +35,8 @@ func printTable(cmd *cobra.Command, plugin connectcustompluginv1.ConnectV1Custom
 	table := output.NewTable(cmd)
 	table.Add(&flinkArtifactOut{
 		Name:          plugin.GetDisplayName(),
-		PluginId:      plugin.GetId(),
-		VersionId:     plugin.GetConnectorClass(),
+		Plugin:        plugin.GetId(),
+		Version:       plugin.GetConnectorClass(),
```

Comment:
@byang-confluent, are you ok with the CLI team making this change to improve standardization across our commands?

I'm also curious why "connector class" maps to "version ID". These seem like two separate things.

Reply:
this is a temporary workaround that versioning api is not GA yet, but flink udf requires this information. 

we are working on versioning api this quarter to deliver this. and update cli after that. please keep this setting for now.

Reply:
reverting this back then

Reply:
> please keep this setting for now.

We're not deleting it, we're just asking if we can change the field names from "Plugin ID" to "Plugin" and "Version ID" to "Version".

Reply:
yes, that should OK